content/docs/user-guide/assets/asset-processor/interface.md

```diff
@@ -46,9 +46,9 @@ Jobs in the Asset Status list display the statuses below:
 * {{< icon "pending.svg" >}} **Pending** - The job has been created, but has not run.
 * {{< icon "processing.svg" >}} **Processing** - The job is in progress.
 * {{< icon "warning-yellow.svg" >}} **Completed - Warning** - The job has completed but has emitted a warning.
-* {{< icon "error.svg" >}} **Completed - Error** - The job has completed and has emitted an error.
+* {{< icon "warning-yellow.svg" >}} **Completed - Error** - The job has completed and has emitted an error.
```

Comment:
Are these the right icons? We did update to the correct icons in one of these docs.